.gitignore

```diff
@@ -11,3 +11,10 @@ __pycache__
 .env
 .private
 *.tfvars
+
+# tardigrade-ci
+.tardigrade-ci
```

Comment:
You're ignoring and committing the file?  Or maybe commit by mistake?

Reply:
oops, wasn't supposed to commit that, it's pulled in automatically by the make target